### ISSUE TITLE
Expand on "The distinction between an incentive and a program". TODO

### ISSUE THREAD

**Issue by FredericoAndrade** [NODE-Collective/specification#2]
https://github.com/NODE-Collective/specification/blob/1e5595660f329a6259f4f7e27f43cff2e39d4e4e/SPECIFICATION.md?plain=1#L35

**Comment by kswb** [comment]
We used the following thought process for distinguishing between programs and incentives.

TL;DR - Programs are rectangles and incentives are squares in that as programs get larger and more complex, they diverge from the incentives contained within, but smaller programs and their incentives may be indistinguishable.

- A program typically has an explicit budget, formal documentation, and support team that collectively manage one or more incentives. Commonly, larger programs have handbooks or manuals that are regularly updated which lay out terms and conditions, resources, and other information.
- An incentive typically resides within the program structure, inheriting the requirements, support team, and other artifacts of program management. In smaller organizations or within smaller programs, there may not be any distinction between the two because of the scope. Larger programs often 'shard' out incentives with budget allocations (think [California Energy Smart Homes](https://caenergysmarthomes.com/)' MANY child incentives within both the Major Alterations program and New Construction programs). For this, we called ESH New Alterations and New Construction programs (two in total) while the incentives were distinct and had child relationships to those programs as I described above.
